google_cdn-external/main.tf

```diff
@@ -55,9 +55,13 @@ resource "google_compute_backend_service" "default" {
       serve_while_stale            = lookup(var.cdn_policy, "serve_while_stale", null)
       signed_url_cache_max_age_sec = lookup(var.cdn_policy, "signed_url_cache_max_age_sec", null)
       cache_key_policy {
-        include_host         = lookup(var.cache_key_policy, "include_host", true)
-        include_protocol     = lookup(var.cache_key_policy, "include_protocol", true)
-        include_query_string = lookup(var.cache_key_policy, "include_query_string", true)
+        include_host           = lookup(var.cache_key_policy, "include_host", true)
+        include_protocol       = lookup(var.cache_key_policy, "include_protocol", true)
+        include_query_string   = lookup(var.cache_key_policy, "include_query_string", true)
+        include_http_headers   = lookup(var.cache_key_policy, "include_https_heaers", null)
```

Comment:
typo on this line?

Reply:
"heaers"

Reply:
also the var is include_HTTP_headers while the lookup is for include_HTTPS_...

Reply:
(maybe that last one is intentional?)

Reply:
Just a typo! Thanks for catching